packages/app/models/stack-order/fetcher.ts

```diff
@@ -0,0 +1,12 @@
+import { getCowOrders } from "@/models/cow-order";
+import { Order } from "@/models/order";
+import { StackOrder } from "@/models/stack-order/types";
+
+export async function getStackOrders(orders: Order[]): Promise<StackOrder[]> {
+  const ordersPromises = orders.map(async (order): Promise<StackOrder> => {
+    const cowOrders = await getCowOrders(order.id);
```

Comment:
Just curious, what extra info are we getting from CoW API ?

Reply:
I don't know what you mean by extra. But I'll explain the data we need from cow:

We are getting the cow orders, we need to `executedBuyAmount`, `executedSellAmount` to know how much funds we're used to buy (eg for avg price) also how much you have stacked (total of the executedBuyAmount) and stuff like that.

It's info related to a transaction like this on cow:

https://explorer.cow.fi/gc/orders/0x4a25a005f8f06231a0fefde069150c9a0fdb9fae2983b3fa341d85d9b49d336068b57d8b652aee685c6b8c387616911eca5ed883649dca24

here is an example:
```json
{
    "creationDate": "2023-06-29T17:47:12.693428Z",
    "owner": "0x68b57d8b652aee685c6b8c387616911eca5ed883",
    "uid": "0x4a25a005f8f06231a0fefde069150c9a0fdb9fae2983b3fa341d85d9b49d336068b57d8b652aee685c6b8c387616911eca5ed883649dca24",
    "availableBalance": null,
    "executedBuyAmount": "190625403172204122069",
    "executedSellAmount": "13783979663205039289",
    "executedSellAmountBeforeFees": "13783979663205039289",
    "executedFeeAmount": "0",
    "invalidated": false,
    "status": "fulfilled",
    "class": "limit",
    "surplusFee": "716753416401807",
    "surplusFeeTimestamp": "2023-06-29T17:47:15.755532Z",
    "executedSurplusFee": "716753416401807",
    "settlementContract": "0x9008d19f58aabd9ed0d60971565aa8510560ab41",
    "fullFeeAmount": "716753416401807",
    "solverFee": "716753416401807",
    "isLiquidityOrder": false,
    "fullAppData": null,
    "sellToken": "0xe91d153e0b41518a2ce8dd3d7944fa863463a97d",
    "buyToken": "0x177127622c4a00f3d409b75571e12cb3c8973d3c",
    "receiver": "0x05a4ed2367bd2f0aa63cc14897850be7474bc722",
    "sellAmount": "13783979663205039289",
    "buyAmount": "1",
    "validTo": 1688062500,
    "appData": "0x9b5c6dfa0fa4be89e17700f05bee8775b281aa6d2dac7dfbf3945e0f9642d777",
    "feeAmount": "0",
    "kind": "sell",
    "partiallyFillable": false,
    "sellTokenBalance": "erc20",
    "buyTokenBalance": "erc20",
    "signingScheme": "eip1271",
    "signature": "0x000000000000000000000000e91d153e0b41518a2ce8dd3d7944fa863463a97d000000000000000000000000177127622c4a00f3d409b75571e12cb3c8973d3c00000000000000000000000005a4ed2367bd2f0aa63cc14897850be7474bc722000000000000000000000000000000000000000000000000bf4a8879188b7cb9000000000000000000000000000000000000000000000000000000000000000100000000000000000000000000000000000000000000000000000000649dca249b5c6dfa0fa4be89e17700f05bee8775b281aa6d2dac7dfbf3945e0f9642d7770000000000000000000000000000000000000000000000000000000000000000f3b277728b3fee749481eb3e0b3b48980dbbab78658fc419025cb16eee34677500000000000000000000000000000000000000000000000000000000000000005a28e9363bb942b639270062aa6bb295f434bcdfc42c97267bf003f272060dc95a28e9363bb942b639270062aa6bb295f434bcdfc42c97267bf003f272060dc9",
    "interactions": {
        "pre": [],
        "post": []
    }
}
```

Reply:
Ok, got it! Thanks